apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -49,7 +49,7 @@ const ProgressBubbleSet = React.createClass({
   propTypes: {
     levels: PropTypes.arrayOf(
```

Comment:
this should be `PropTypes.arrayOf(levelType)` where levelType is imported from progressTypes I believe.

PropTypes.levelType will presumably just be undefined.